zerver/openapi/zulip.yaml

```diff
@@ -9954,7 +9954,7 @@ paths:
         - name: email
           in: path
           description: |
-            The email address of the user whose details you want to fetch.
+            The Zulip API email address of the user whose details you want to fetch.
```

Comment:
Even though this wasn't labeled as "Zulip display email", we discussed [here on CZO](https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/api.20send.20message.20using.20mail.20addresses/near/1566875) that it in fact is.